PowerArubaSW/Public/RestTimeout.ps1

```diff
@@ -0,0 +1,95 @@
+
+#
+# Copyright 2018, Alexis La Goutte <alexis.lagoutte at gmail dot com>
+# Copyright 2018, Cédric Moreau <moreaucedric0 at gmail dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+function Get-ArubaSWRestSessionTimeout {
+
+    <#
+        .SYNOPSIS
+        Get Session Timeout when you connect to a switch
+
+        .DESCRIPTION
+        Get Session Timeout
+
+        .EXAMPLE
+        Get-ArubaSWRestSessionTimeout
+        This function give you idle time before being disconnected
+
+    #>
+
+    Begin {
+    }
+
+    Process {
+
+        $url = "rest/v4/session-idle-timeout"
+
+        $response = invoke-ArubaSWWebRequest -method "GET" -url $url
+
+        $run = ($response | convertfrom-json).timeout
+
+        $run
+
+    }
+
+
+    End {
+    }
+}
+
+
+
+function Set-ArubaSWRestSessionTimeout {
+
+    <#
+        .SYNOPSIS
+        Set Session Timeout when you connect to a switch
+
+        .DESCRIPTION
+        Set Session Timeout
+
+        .EXAMPLE
+        Set-ArubaSWRestSessionTimeout <seconds>
```

Comment:
Add direclty the value for example Set-ArubaSWRestSessionTimeout 1200
and on description set it is on second

Reply:
don't forget this change too